### PR TITLE
Fixing `PostProcessManager.export_to_vtk`

### PR DESCRIPTION
**Pull request by anlavandier**
This PR fixed `PostProcessManager.export_to_vtk` which did not do anything.
It consists of:
*  Putting starred expressions where they need to be
* Adding a `debug` argument to `export_to_vtk` that returns what is computed in the function. 
* Adding `assert` statements to check if the right fields are exported.

### CODE REVIEW

File: psydac/api/tests/test_postprocessing.py [hunk 1]
```diff
@@ -47,6 +47,7 @@ def test_OutputManager():
 
     try:
         Om.export_fields(uh_static=uh)
+        raise ValueError
     except AssertionError:
         pass
```
Comment (yguclu) [inline psydac/api/tests/test_postprocessing.py:52]: Is the purpose of this piece of code to check that an assertion is raised? If so, there is a specific way to do so with Pytest:

```suggestion
    with pytest.raises(AssertionError):
        Om.export_fields(uh_static=uh)
```
If the correct exception is not raised, the test will fail.

File: psydac/api/postprocessing.py [hunk 2]
```diff
@@ -909,7 +916,7 @@ def export_to_vtk(self, filename_pattern, grid, npts_per_cell=None, snapshots='n
 
         smart_eval_dict = {}
         for i, snapshot in enumerate(snapshots):
-            self.load_snapshot(snapshot, tuple(fields.keys()))
+            self.load_snapshot(snapshot,*fields.keys())
```
Comment (yguclu) [inline psydac/api/postprocessing.py:919]: Missing empty space after the comma

File: psydac/api/postprocessing.py [hunk 3]
```diff
@@ -791,32 +789,36 @@ def load_snapshot(self, n, *fields):
...
             Number of leading zeros in the time indexing of the files. 
             Only used if ``snapshot`` is not ``'none'``. 
 
-        fields: dict
+        fields : dict
             Dictionary with the fields to export as keys and the name under which to export them as values
+        
+        debug : bool, default=False
+            If true, returns ``(mesh, pointData_list)`` where ``mesh`` is ``(x_mesh, y_mesh,  z_mesh)``
+            and ``pointData_list`` is the list of all the pointData dictionnaries.
```
Comment (yguclu) [inline psydac/api/postprocessing.py:821]: Typo: dictionnaries -> dictionaries

**Review by yguclu — CHANGES_REQUESTED** [review]

File: psydac/api/postprocessing.py (hunk 2 above)
Comment (anlavandier) [reply psydac/api/postprocessing.py:919]: Fixed in 3e32950

**Review by yguclu — APPROVED** [review]

**Pull request merged**